charts/zitadel/acceptance_test/config_test.go

```diff
@@ -35,7 +35,10 @@ var (
 	Postgres = databaseChart{
 		repoUrl: "https://charts.bitnami.com/bitnami",
 		name:    "postgresql",
-		version: "12.10.0",
+		testValues: map[string]string{
+			"image.repository":                   "bitnamilegacy/postgresql",
+			"volumePermissions.image.repository": "bitnamilegacy/os-shell",
+		},
```

Comment:
Would it not be better to change the example values like in https://github.com/zitadel/zitadel-charts/blob/main/examples/1-postgres-insecure/postgres-values.yaml
And also to upgrade the pinned version instead of using latest, to have deterministic behavior?